src/DataViewer.js

```diff
@@ -0,0 +1,93 @@
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import React, { Component } from 'react';
+import ReactJson from 'react-json-view'
+import Typography from '@material-ui/core/Typography';
+import firebase from 'firebase';
+import Paper from 'material-ui/Paper';
```

Comment:
Can you find an alternative to using the deprecated `<Paper>` Material UI element? I just removed them from the other components.

Reply:
Yup!